docs/GettingStarted.md

```diff
@@ -0,0 +1,95 @@
+To help you get along with Maestro, by the end of this section you should have a scheduler up and running.
+
+## Prerequisites
+- Golang v1.17+
+- Linux/MacOS environment
+- Docker
+
+### Clone Repository
+Clone the [repository](https://github.com/topfreegames/maestro) to your favorite folder.
+
+## Building and running locally
+> For this step, you need docker running on your machine.
+
+In the folder where the project was cloned, simply run:
+
+```shell
+make maestro/start
+```
+
+This will build and start all containers needed by Maestro, such as databases and maestro-modules. 
+Because of that, be aware that it might take some time to finish.
+
+## Find rooms-api address
+To simulate a game room, it's important to find the address of running **rooms-api** on the local network.
+
+To do that, with Maestro containers running, simply use:
+
+```shell
+docker inspect -f '{{range.NetworkSettings.Networks}}{{.Gateway}}{{end}}' maestro_rooms-api_1
```

Comment:
I think that is good to point out here that the name of the container could change. It shouldn't  always be `maestro_rooms-api_1`